components/Storyblok/partials/SbLink.tsx

```diff
@@ -65,9 +66,17 @@ export const SbLink = React.forwardRef<HTMLAnchorElement, SbLinkProps>((props, r
   // Story or Internal type link.
   // ---------------------------------------------------------------------------
   if (props.link.linktype === 'story') {
+    // If the internal link already starts with a slash (eg, WYSIWYG inline internal links), remove it.
+    if (linkUrl.startsWith('/')) {
+      linkUrl = linkUrl.substring(1);
+    }
```

Comment:
The `href` of internal links added in WYSIWYG has an extra `/` in front compared to the `cached_url` of a Storyblok link field outside of a WYSIWYG 😮‍💨